src/main/java/database/exception/exception/CreationDatabaseException.java

```diff
@@ -0,0 +1,8 @@
+package database.exception.exception;
+
+public class CreationDatabaseException extends BadRequestException {
```

Comment:
In my understanding this exception can occur not only when user wants to create already existing database. I would return 500 if this exception happened.